core/build.gradle.kts

```diff
@@ -56,6 +55,7 @@ dependencies {
     testCompile(TestLibraries.testContainers)
     testCompile(TestLibraries.testContainersInflux)
     testImplementation(TestLibraries.mockitoKotlin)
+    testImplementation(TestLibraries.koin)
```

Comment:
Do we really need koin for tests?

Reply:
Yeah, we need now to inject analytics in every scenario tests by using `fun Spec.initKoin()`

Reply:
I've tried to remove it as much as possible from tests